exercises/palindrome-products/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(Debug, PartialEq)]
+pub enum Error{
+    Empty,
+    RangeFailure,
+}
+#[derive(Debug, PartialEq)]
+pub struct ReturnVals{
```

Comment:
This should be named `Palindrome`, which is much more descriptive than `ReturnVals`.